libraries/lib-datahandler/src/main/java/org/eclipse/sw360/datahandler/common/SW360Utils.java

```diff
@@ -197,6 +239,26 @@ public static String printFullname(Release release) {
         return getReleaseFullname(vendorName, release.getName(), release.getVersion());
     }
 
+    public static Map<Todo, TodoInfo> getProjectObligations(Project project) {
```

Comment:
just a quick question regarding the naming: is todo and obligation the same or not?

Reply:
A todo is an obligation, but not the other way around.